### ISSUE TITLE
Windows contents not drawn while resizing

### ISSUE THREAD

**Issue by magicgoose** [PistonDevelopers/conrod#958]
For all 4 examples (all_winit_glium, canvas, primitives, text), when I resize the window, the window contents are not drawn (there's only solid white fill) until the mouse button is released.

Tested on latest macOS, with HiDPI display.

```
$ git rev-parse --short=1 HEAD  # latest commit at the time of writing
acb6

$ cargo --version
cargo 0.16.0-dev (6e0c18c 2017-01-27)

$ rustc --version
rustc 1.15.1
```

I used commands from `http://docs.piston.rs/conrod/conrod/guide/chapter_2/index.html#installing-rust-and-cargo` to run examples.

**Comment by mitchmindtree** [comment]
Thanks for the issue!

This is a well known problem with older `winit` versions which conrod's `winit` feature currently depends on. This has since [been fixed](https://github.com/tomaka/winit/pull/132) in `winit` 0.6 (along with loads of other fixes) and I'm currently [attempting to update glutin](https://github.com/tomaka/glutin/pull/864) so that we can in turn update `glium` and both conrod's `winit` and `glium` features. This is currently my main priority and I'm hoping to get through it within the next few days.

**Comment by mitchmindtree** [comment]
I believe this should be fixed as of #1028. There are still some quirks with the macos resizing behaviour where once resizing begins, a call to `run_forever` or `poll_events` will not return or awaken until the resize is complete. There is an incredible [fix for this in the works](https://github.com/tomaka/winit/pull/237) by willglynn, but we need more macos reviewers looking at the PR as the changes are pretty significant and intricate.
